superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -476,6 +476,18 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  rowsReturned() {
+    return (
+      <div className="ReturnedRows">
+        {this.state.data && this.state.data.length >= 0 && (
```

Comment:
you can use optional chaining here. Can the length ever be less than 0?

Reply:
I think I could actually just remove the conditions, since a tighter version of it happens in the render.